[FIX] theme_avantgarde : colour variable fix

karlosvalverde opened the pull request:
[FIX] theme_avantgarde : colour variable fix

This commit fixes the previous hardcoded assignation to colour
customisations in theme Avantgarde's snippets, by using proper
colour variables, which allow the snippet customisations change
dynamically.

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/design-themes/pull/509).

qsm-odoo commented:
@karlosvalverde Stumbled upon this, should this be closed or rebased/retargeted ?

robodoo commented:
This PR targets the disabled branch odoo/design-themes:saas-14.5, it can not be merged.

karlosvalverde commented:
@qsm-odoo Could you have a look into this PR? Thanks!

karlosvalverde commented:
I just changed the target of the PR to `15.0` instead of `master` and rebased the commit on the top of `15.0`. If you think there is something else to check, let me know 😉

karlosvalverde commented:
Could you have a look into this PR @qsm-odoo ?

Thanks! 👍

qsm-odoo reviewed (approved):
@robodoo r+
